impl/src/main/java/impl/UserResourceImpl.java

```diff
@@ -92,8 +94,21 @@ public UserResourceImpl(UserDao userDao, ResponseHeaderHolder responseHeaderHold
 
     private void addAsCookie(final ApplicationToken applicationToken, User user) {
         Map<String, Object> headers = new HashMap<>();
-        headers.put("Set-Cookie", "application=" + applicationToken.getApplicationToken() + "; path=/; domain=" + applicationTokenConfig.getDomain() + ";");
+        headers.put("Set-Cookie", getCookie(applicationToken.getApplicationToken(), SESSION_MAX_AGE_SECONDS, applicationTokenConfig.getDomain()));
         responseHeaderHolder.addHeaders(user, headers);
     }
 
+    @Override
+    public Observable<Long> signOut(Auth auth) {
```

Comment:
Do we need an auth here? If we redirect to signOut on 401's that could maybe lead to infinite loops?

Reply:
nope we do not redirect to signout :)